### PR TITLE
Sort targets in Xcode project

### PR DESCRIPTION
**Pull request by beefon**
This PR activates sorting of the targets in generated Xcode project, which makes the navigation around the code much easier.

pbx

### CODE REVIEW

File: Sources/Xcodeproj/pbxproj().swift [hunk 1]
```diff
@@ -272,7 +272,8 @@ func xcodeProject(
     // source directory of each target to the corresponding group, so that
     // source files added later will be able to find the right group.
     @discardableResult
-    func createSourceGroup(named groupName: String, for targets: [ResolvedTarget], in parentGroup: Xcode.Group) -> Xcode.Group? {
+    func createSourceGroup(named groupName: String, for resolvedTargets: [ResolvedTarget], in parentGroup: Xcode.Group) -> Xcode.Group? {
+        let targets = resolvedTargets.sorted { (left, right) -> Bool in left.name < right.name }
```
Comment (aciidb0mb3r) [inline Sources/Xcodeproj/pbxproj().swift]: You can just do: `let targets = targets.sorted({ $0.name < $0.name })`

Comment (beefon) [reply Sources/Xcodeproj/pbxproj().swift]: 👍

**Comment by dimorinny** [comment]
Approved

File: Package.resolved [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "state": {
+          "branch": "master",
+          "revision": "0615bdec228d6ed5be4ffa9ae00de0c5e0cb937c",
+          "version": null
+        }
+      }
+    ]
+  },
+  "version": 1
+}
```
Comment (artyom-razinov) [inline Package.resolved]: Maybe to rebase to remove this change?

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

**Pull request merged**